docs/includes/openapi/openapi-ui.adoc

```diff
@@ -46,7 +46,11 @@ ifndef::rootdir[:rootdir: {docdir}/../..]
 
 SmallRye offers an link:{smallrye-openapi-ui-base-url}[OpenAPI user interface component] which displays a web page based on your application's OpenAPI document.
 Through that UI, users can invoke the operations declared in the document.
-While not generally suitable for end-users, the OpenAPI UI can be useful for demonstrating and "test driving" your service's endpoints.
+
+[NOTE]
```

Comment:
In the past I was asked to include this blurb for experimental features. Would this note fit under that category?  (example below)

=== Experimental
WARNING: Helidon Vault support is still experimental and not intended for production use. APIs and features have not yet been fully tested and are subject to change.